FA22.P05.Web/ClientApp/src/navigation/Navbar.tsx

```diff
@@ -19,9 +27,14 @@ export default function Navbar() {
 
   return (
     <>
-      <AppBar position="static" sx={{ background: "#1f2b39" }}>
```

Comment:
change the color back to this hex value for me please

Reply:
done